src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -105,11 +105,27 @@ def proxy(self, method, path, params=None, json_response=True):
 @ApiController('/rgw/bucket', Scope.RGW)
 class RgwBucket(RgwRESTController):
 
+    def _append_bid(self, bucket):
+        """
+        Append the bucket identifier that looks like [<tenant>/]<bucket>.
+        See http://docs.ceph.com/docs/jewel/radosgw/multitenancy/ for
```

Comment:
Nit: would it make sense to point to a more recent version of the docs here?

Reply:
Indeed, that's a good idea :-)